Source/WebKit/UIProcess/ios/WKScrollView.mm

```diff
@@ -549,10 +552,31 @@ - (CGPoint)_puic_contentOffsetForCrownInputSequencerOffset:(double)sequencerOffs
 #endif // HAVE(PEPPER_UI_CORE)
 
 #if ENABLE(OVERLAY_REGIONS_IN_EVENT_REGION)
+- (bool)_updateOverlayRegions:(const Vector<CGRect> &)overlayRegions
+{
+    if (_overlayRegions == overlayRegions)
+        return false;
+
+    [self willChangeValueForKey:@"overlayRegions"];
+    _overlayRegions = overlayRegions;
+    [self didChangeValueForKey:@"overlayRegions"];
+    return true;
+}
+
+- (const Vector<CGRect> &)_overlayRegions
```

Comment:
I don't love having the two getters with the different types just differentiated by a leading underscore. Anything we can put in the name to differentiate?

Reply:
(or, better, can we just make the NSArray when they're set and only have one getter?)

Reply:
Oh I don't even use the `_overlayRegions` version, I should just remove that